user-interface/2001-STKE-staking.md

```diff
@@ -1 +1,127 @@
-# Staking
\ No newline at end of file
+# Staking
+
+Staking is the act of securing a Vega network by nominating good validators with the [governance token](../protocol/0071-STAK-erc20_governance_token_staking.md). Staking is rewarded with a share of trading fees (and [treasury rewards](../0056-REWA-rewards_overview.md)). See the [glossary](../glossaries/staking-and-governance.md) and [these specs](../protocol#delegation-staking-and-rewards) for more on staking.
+
+When staking a user may be motivated to select validators to maximize the rewards they get for the tokens they hold, this means selecting validator(s) who are less likely to be penalized (e.g. over staked, poor performance). Users may wish to stake more than one validator to diversify. Users will want/need to manage their stake over time to ensure they are getting a good return, e.g. move stake between validators. Staking is also important for facilitating protocol upgrades.
+
+## Understand staking on Vega
+
+When considering whether to stake on Vega, I...
+
+- **must** see information to help inform me what return I might expect from staking (other protocols might show a typical APY) <a name="1002-STKE-001" href="#1002-STKE-001">1002-STKE-001</a>
```

Comment:
Should all the staking codes now start with `2001-`?

Reply:
resolved in https://github.com/vegaprotocol/specs/pull/1301/commits/6c3833c5fe08ba0fd8d214c1dddd0eced8c6f4d1